addons/l10n_mx/migrations/11.0.2.1/pre-migration.py

```diff
@@ -0,0 +1,21 @@
+from odoo import SUPERUSER_ID, api
+
+
+def mx_update_account_type(cr):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        account = env.ref('l10n_mx.cuenta801_01', False)
+        if account:
+            account.write({'user_type_id': env.ref('account.data_account_type_expenses').id})
+        account = env.ref('l10n_mx.cuenta108_01', False)
+        if account:
+            account.write({'user_type_id': env.ref('account.data_account_type_current_assets').id})
+        account = env.ref('l10n_mx.cuenta108_02', False)
+        if account:
+            account.write({'user_type_id': env.ref('account.data_account_type_current_assets').id})
```

Comment:
the IF must check tha count is called indetical as before, if yuser changed the code the xmlid will be the same but the meaning of the code change then check:

tag_ids.contains >> XXX.YYY
code.startsWith(XXX.YYY)

If account where changes this migration is not Ok this must be ignored.

cc @luistorresm

Reply:
@nhomar 

I update, to ensure that the old account type is the type that was changed in the commit https://github.com/odoo/odoo/commit/632cc5417796ff375d47767fd7cdb358a5d4ce94

What do you think?

Regards